utils/flatmap.go

```diff
@@ -14,13 +14,14 @@ const UnknownVariableValue = "74D93920-ED26-11E3-AC10-0800200C9A66"
 func Expand(m map[string]string, key string) interface{} {
 	// If the key is exactly a key in the map, just return it
 	if v, ok := m[key]; ok {
-		if v == "true" {
+		switch v {
+		case "true":
 			return true
-		} else if v == "false" {
+		case "false":
 			return false
+		default:
+			return v
 		}
```

Comment:
unrelated change for this PR: with that said, this seems to be a use case for `strconv.ParseBool`

Reply:
On further review, there's no need for this package to begin with. The functions aren't really being called with the sole exception of pointers and those are fungible with ptr.To. The right solution is to remove the package entirely.